package.json

```diff
@@ -8,9 +8,9 @@
   "files": ["dist"],
   "type": "module",
   "scripts": {
-    "build": "tsc",
+    "build": "tsc --noEmit && vite build",
```

Comment:
Discussion point: should we keep building with vite? Or just revert to `tsc`?

Only benefit I can give for building with vite is that it minifies the output, but it requires hacking `chalk` to force run in color mode (see `vite.config.ts`)